lms/tests.py

```diff
@@ -28,4 +28,6 @@ def test_api_docs(self):
         Tests that requests to the `/api-docs/` endpoint do not raise an exception.
         """
         response = self.client.get('/api-docs/')
-        self.assertEqual(200, response.status_code)
+        # Appsembler: Disabled for security reasons. Should be back after upgrading drf-yasg. See RED-2861 for details
+        # self.assertEqual(200, response.status_code)
+        self.assertEqual(404, response.status_code)
```

Comment:
groups the feature flag pieces and tests together


```suggestion
        self.assertFalse(settings.FEATURES.get('TAHOE_ENABLE_API_DOCS_URLS'))
        self.assertEqual(404, response.status_code)  # Tahoe: Changed from `200`
```